.tooling/eslint/custom-plugin.js

```diff
@@ -10,6 +10,7 @@ export const customPlugin = {
 					const isDS = path.startsWith('@ds/') || path.includes('../src-ds/')
 					const isI18n = path.startsWith('@i18n/') || path.includes('../src-i18n/')
 					const isUtils = path.startsWith('@utils/') || path.includes('../src-utils/')
+					const isAppLib = path.startsWith('@app/library') || path.includes('../src-app/library/')
```

Comment:
`path.includes('/library/')` ?

Reply:
That's too generic, it might cause issues.